### DISCUSSION TITLE
Lost eth trying to fix problem

### DISCUSSION THREAD

**Discussion by Buttaz95** [wormhole-foundation/wormhole#522] [Problems / bugs] [answered]
Hi there ive ended up spending all my money trying to solve this problem I originally wanted to bridge my eth to solana phantom wallet it arrived as an NFT . I couldn't find anywhere to swap it for sol and saw the only thing I could swap it for was allbridge wrapped eth. I read somewhere about reversing the process through sending the eth back through the wormhole into my metamask and now it's not even recognised as any type of eth I've spent nearly 200Gbp in gas trying to fix it and I've gotten knowhere please help. (On website)

**Comment by leoluk** [comment, marked as answer]
Phantom bug, should be fixed by now.
